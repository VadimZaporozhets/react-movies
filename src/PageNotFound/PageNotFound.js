import React from 'react';
import { Link } from 'react-router-dom';
import { object } from 'prop-types';

import { routesPaths } from '../routes';

export const PageNotFound = ({ staticContext = {} }) => {
    staticContext.notFound = true;

    return (
        <>
            <Link to={routesPaths.HOME}>Go to Home Page</Link>
            <h1>Page not found</h1>
        </>
    );
};

PageNotFound.propTypes = {
    staticContext: object
};
