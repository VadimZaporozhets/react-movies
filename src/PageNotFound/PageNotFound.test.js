import React from 'react';
import { shallow } from 'enzyme';

import { PageNotFound } from './PageNotFound';

describe('PageNotFound', () => {
    it('should render PageNotFound component', () => {
        const wrapper = shallow(<PageNotFound />);

        expect(wrapper).toMatchSnapshot();
    });
});
