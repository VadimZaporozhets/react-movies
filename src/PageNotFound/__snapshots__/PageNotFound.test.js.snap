// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should render PageNotFound component 1`] = `
<Fragment>
  <Link
    to="/"
  >
    Go to Home Page
  </Link>
  <h1>
    Page not found
  </h1>
</Fragment>
`;
